src/Exchange_Rates.json

```diff
@@ -0,0 +1,23 @@
+{
+  "success": true,
+  "base": "USD",

```

Comment:
baseCurrency

Reply:
Hello @ronehr-microsoft, this is the format as returned by the Forex API in its JSON response. 

So I am using the same JSON model for the cache file as well as, for the JSON response from API.

Should I create 2 different JSON models one for the API response and one for the cached file?

Reply:
That's fine. Let's keep the model consistent, adding a transformer increases complexity.